server/base/src/test/java/org/apache/accumulo/server/util/TabletIteratorTest.java

```diff
@@ -186,7 +177,7 @@ private Scanner createMockScanner(final TreeMap<Key,Value> data) {
       @Override
       public Iterator<Entry<Key,Value>> answer() throws Throwable {
         Iterator<Entry<Key,Value>> iter = data.entrySet().iterator();
-        iter = Iterators.filter(iter, state.getScanPredicate());
+        iter = Iterators.filter(iter, state::getScanPredicate);
```

Comment:
@ctubbsii I am not sure why but Java can't find the test lambda when written this way...it gives "incompatible types not convertible to boolean" compilation error.  I have to call ```state.getScanPredicate()``` first then can pass it as a method reference.  I think its the way we have ScannerState written.

Reply:
Hmm, I thought it worked for me in Eclipse... but I didn't try to build on the command line.

Reply:
Oh wait, never mind. I made a mistake here. Disregard my original suggestion.